interface/wx/arrstr.h

```diff
@@ -409,6 +416,54 @@ int wxDictionaryStringSortAscending(const wxString& s1, const wxString& s2);
  */
 int wxDictionaryStringSortAscending(const wxString& s1, const wxString& s2);
 
+
+    /**
+        Case-sensitive comparison used for Natural Sort functions.
+        Functions in the same way as Cmp(), with the exception that numbers within
+        the string are recognised, and compared numerically, rather than alphabetically.
+        When used for sorting, the result is that e.g. file names containing numbers
+        are sorted in a natural way.
+        
+        e.g. Sorting using Cmp()
+        - file1.txt
+        - file10.txt
+        - file100.txt
+        - file2.txt
+        - file20.txt
+        - file3.txt
+        
+        e.g. Sorting using CmpNatural()
+        - file1.txt
+        - file2.txt
+        - file3.txt
+        - file11.txt
+        - file20.txt
+        - file100.txt
+        
```

Comment:
Could you please add `@since 3.1.2` to this and the other functions docs?

Reply:
Done.